skopt/space.py

```diff
@@ -255,7 +258,8 @@ def rvs(self, n_samples=None, random_state=None):
 
     @property
     def transformed_size(self):
-        return len(self.categories)
+        size = len(self.categories)
+        return size if size != 2 else 1
```

Comment:
no, it'll be 1, see this line https://github.com/scikit-learn/scikit-learn/blob/51a765a/sklearn/preprocessing/label.py#L441